Scripts/Runtime/Core/CollectionsRegistry.cs

```diff
@@ -323,15 +303,50 @@ public void PrepareForEditorMode()
             for (int i = 0; i < collections.Count; i++)
                 collections[i].PrepareForEditorMode();
         }
-
-        public void ClearBadItems()
+#endif
```

Comment:
Not sure why an `#endif` got added here, I don't see a new `#if` or an `#endif` that got removed.

Maybe the diff messed up or something, not sure. I kind of expect that you'd have a compile error if this didn't match so it's probably nothing but it looked weird so I wanted to point it out.

Reply:
Yeah its just how is presented here, it does have the initial #if on line 294